std/string.d

```diff
@@ -4763,6 +4763,15 @@ body
  * in one of a known set of strings, and the program will helpfully
  * autocomplete the string once sufficient characters have been
  * entered that uniquely identify it.
+ *
+ * Params:
+ *      array of strings
```

Comment:
I appreciate the effort of adding Params: and Returns:, but just mentioning the types is hardly helpful.